### ISSUE TITLE
Add a way to make an item a physics sensor.

### ISSUE THREAD

**Issue by dcman** [UnderwaterApps/overlap2d#268]
Can you add a check box or something in PhysicsComponent to make item a sensor.

**Comment by azakhary** [comment]
I am sorry I haven't used that part of box2d ever, will do some reading. but. in short. is it just a boolean? or some other data is required too?

**Comment by dcman** [comment]
Yeah it's just a Boolean in the FixtureDef class. It causes the object to
signal the collision detection. But does not allow the objects to interact
with each other.

https://libgdx.badlogicgames.com/nightlies/docs/api/com/badlogic/gdx/physics/box2d/FixtureDef.html#isSensor

Thanks

**Comment by CoderJoe1999** [comment]
this is definitely needed! It would be much nicer to select this from the editor rather than having to set them to sensor through the code
